packages/lesmis/src/components/NavBar.js

```diff
@@ -18,7 +18,8 @@ import { useUser } from '../api';
 import { useUrlParams } from '../utils';
 
 const Container = styled.nav`
-  position: relative;
+  position: sticky;
+  top: 0;
```

Comment:
Not strictly included in the ticket definition, but seemed to be true from Dave's video that the nav bar is sticky